README.md

```diff
@@ -110,20 +110,25 @@ _Note for Windows Users: You should use a [Bash Shell][bash] to follow the steps
 To compile Kelp from source:
 
 1. [Download][golang-download] and [setup][golang-setup] Golang _v1.13 or later_.
+    * Confirm that `$GOPATH` is set, and that `GOBIN=$GOPATH/bin`
 2. [Install Glide][glide-install] for dependency management
     * `curl https://glide.sh/get | sh`
 3. Install [Yarn][yarn-install] and [NodeJs][nodejs-install] to build the Kelp GUI
 4. Clone the repo into `$GOPATH/src/github.com/stellar/kelp`:
     * `git clone git@github.com:stellar/kelp.git`
-5. Change to the kelp directory and install the dependencies:
+    * Change to `kelp` directory for all following commands: `cd $GOPATH/src/github.com/stellar/kelp`
+5. Install the dependencies:
     * `glide install`
 6. Install the [astilectron-bundler][astilectron-bundler] binary in a folder that is in your `PATH`
+    * `go install` will install this by default to `$GOBIN`, so check that that folder's full filepath is in `$PATH`
 6. Build the binaries using the provided build script (the _go install_ command will produce a faulty binary):
     * `./scripts/build.sh`
-7. Confirm one new binary file:
+7. Confirm one new binary file exists, and optionally run it. 
     * `./bin/kelp`
+    * If you run the binary, check `localhost:8000` after initial setup to confirm that you see the Kelp UI.
 8. Set up CCXT to use an expanded set of priceFeeds and orderbooks (see the [Using CCXT](#using-ccxt) section for details)
     * `sudo docker run -p 3000:3000 -d franzsee/ccxt-rest:v0.0.4`
+    * Note that you need to stop running the `./bin/kelp` binary due to common port bindings
```

Comment:
For Step 7 it is better to suggest running `./bin/kelp version` and ensure there is a reasonable output with version information.

That will not start up the GUI which is already covered in the section on how to Run Kelp (and we will promote better in the README once out of experimental phase). Starting the Kelp GUI starts up CCXT-rest if it's not running on docker etc., which is unnecessarily complicating things in step 8. Added https://github.com/stellar/kelp/issues/487 to address this.

Can remove this Note about conflicting ports once we change to the suggestion of the `version` command
You can also mark this step 8 as (optional). The goal of putting it up here was so that people don't miss the step of how to run CCXT since that was a common question

I think this will allow for a smoother workflow.